CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contributing Guidelines
+
+We love pull requests! Our process for accepting changes has a few steps:
+
+1. If you haven't submitted anything before, and you aren't (yet!) a member of our organization, fork and clone the repo:
+
+        git clone git@github.com:your-username/tomeshnet.github.io.git
+Organization members should clone this repo, instead of working from a fork:
+
+        git clone git@github.com:tomeshnet/tomeshnet.github.io.git
+
+2. Create a new branch for the changes you want to work on. Chose a topic for your branch name that reflects the change:
+
+        git checkout -b <branch-name>
+
+3. Create or modify the files you to make your changes. If you want to show other people work that isn't ready to merge in, commit your changes then [create a pull request (PR)](https://github.com/tomeshnet/tomeshnet.github.io/pull/new/master) with __WIP__ in the title.
+
+4. Once your changes are ready for final review, commit your changes then [create or modify your PR](https://github.com/tomeshnet/tomeshnet.github.io/pull/new/master).
+
+5. Allow others sufficient time for review and comments before merging. For big changes, ping another member to check over what you've done. There may be some fixes or adjustments you'll have to make based on feedback.
```

Comment:
We will probably reference this guide for all Toronto Meshnet repos (rather than writing a set of rules for each). So for big changes, it's probably a good idea to open an issue and tag as **question** to discuss that change before spending the time to make the big change. Then open a PR referencing the issue.